### PR TITLE
Backend login/signup

### PR DESCRIPTION
**Pull request by RDArtemis**
converted java files to jsp and started on firebase connection for login.jsp backend

### CODE REVIEW

File: WebContent/login.jsp
Comment (gretaLiesch) [inline WebContent/login.jsp:1]: Should this file not be called loginback.jsp? Since it provides backend functionality to the login page

**Review by gretaLiesch — COMMENTED** [review]
Looks good overall! I've just added a comment/question about one of the filenames

File: WebContent/login.jsp
Comment (RDArtemis) [reply WebContent/login.jsp:1]: Yes I will update that. I also need to rename the signup to signupback.jsp

**Review by gretaLiesch — APPROVED** [review]
Sweet!

**Pull request merged**